scripts/prepublish.ts

```diff
@@ -0,0 +1,39 @@
+import path from 'path';
+import workspacesRun from 'workspaces-run';
+import { readJSON, writeJSON } from './utils/json';
+
+/** Before publish, let's replace all the relative file links with versions */
+async function main()
+{
+    const { version } = await readJSON<{version: string}>(path.join(process.cwd(), 'package.json'));
+
+    // Update all the packages with versions
+    await workspacesRun({ cwd: process.cwd() }, async (workspace) =>
+    {
+        workspace.config.version = version;
```

Comment:
```suggestion
        if ('version' in workspace.config)
        {
            console.error(
                `Error: ${workspace.dir}/package.json should not have a "version" property. `
                + `Remove it before publishing.`);

            process.exit(1);
        }
        workspace.config = { name: workspace.config.name, version, ...workspace.config };
```

This places the `"version"` right after the `"name"` property, making the published `package.json` a little prettier, otherwise the `"version"` property would be placed at the end of `package.json`.

<img src="https://user-images.githubusercontent.com/8724868/216376482-33fc4dfd-8f53-4a87-8431-e57865bad306.png" width="400px"/>